26_CentralLimitBoxBlur/app_resources/descriptors.hlsl

```diff
@@ -0,0 +1,51 @@
+#include "nbl/builtin/hlsl/central_limit_blur/common.hlsl"
+
+[[vk::binding( 0, 0 )]] Texture2D<nbl::hlsl::float32_t4> input;
+[[vk::binding( 1, 0 )]] RWTexture2D<nbl::hlsl::float32_t4> output;
+
+
+struct BufferAccessor
+{
+	uint32_t2 chosenAxis;
```

Comment:
make life easier for the compiler, make the Chosen Axis a template parameter, and everything will unroll

also no point making it a weird vec2, just a simple 0=x, 1=y, 2=z